codespell_lib/data/dictionary.txt

```diff
@@ -32229,16 +32229,26 @@ lveo->love
 lvoe->love
 Lybia->Libya
 maake->make
+maange->manage
```

Comment:
Perhaps `mange` too, although this is a rare word.

Reply:
I couldn't find this typo.